torchtune/training/quantization.py

```diff
@@ -43,6 +48,22 @@
         Int8DynActInt4WeightQATQuantizer,
     )
 
+from torchtune.utils._version import torch_version_ge
+
+
+_SUPPORTS_INT8_MIXED_PRECISION_TRAINING = (
+    torch_version_ge("2.4.0")
```

Comment:
nit: can remove this check now (was probably more relevant when you first opened the PR)